docs/markdown/Cython.md

```diff
@@ -31,3 +31,30 @@ py.extension_module(
     dependencies : dep_py,
 )
 ```
+
+## C++ intermediate support
+
+*(New in 0.60.0)*
+
+An option has been added to control this, called `cython_language`. This can be
+either `'c'` or `'cpp'`.
+
+For those coming from setuptools/distutils, they will find two things. First,
+meson ignores `# distutils: language = c++` inline directives. Second that Meson
+allows options only on a per-target granularity. This means that if you need to mix
+cython files being transpiled to C and to C++ you need two targets:
+
+```meson
+cython_cpp_lib = static_library(
+    'helper_lib',
+    'foo_cpp.pyx',
+    override_options : ['cython_language=cpp'],
+)
+
+py.extension_module(
+    'foo',
+    'foo.pyx',
```

Comment:
… and here `# this .pyx file will be translated as C`
Since the default is used for the second file, this is not obvious on the first glance what is happening.